ros_ign_gazebo/launch/ign_gazebo.launch

```diff
@@ -10,6 +10,6 @@
     name="ign_gazebo"
```

Comment:
Can't leave a comment on an unchanged line, but `pkg` here needs to be updated to `ros_ign_gazebo`